waiverdb/templates/permissions.html

```diff
@@ -14,28 +14,25 @@
                         <th>Name</th>
                         <th>Maintainers</th>
                         <th>Testcases</th>
-                        <th>Users/Groups</th>
+                        <th>Groups</th>
+                        <th>Users</th>
+                        <th>Description</th>
                     </tr>
                 </thead>
                 <tbody>
                     {% for p in permissions -%}
                     <tr>
                         <td>{{ p["name"] }}</td>
-                        <td>{{ p["maintainers"] | map("urlize") | join(" ") | safe }}</td>
-                        <td>{{ p["testcases"] | join("<br />") | replace('-', '&#8209;') | safe }}</td>
                         <td>
-                            {% if p["groups"] | length > 0 -%}
-                            <span class="font-weight-bold">Groups:</span><br />
-                            {{ p["groups"] | join("<br />") | replace('-', '&#8209;') | safe }}
```

Comment:
Without these replaces, your names will be wrapped by "-" and you'll get a mess of broken lines.
Compare
```
baseos‑ci.*.redhat‑container‑image.*
baseos‑ci.redhat‑container‑image.*
```
and
```
baseos‑ci.*.redhat‑
container‑image.*
baseos‑ci.redhat‑
container‑image.*
```
The last one is worse readable than the first one.

Reply:
Good point. Fixed by using `<ul>`.